src/pages/home/report/ReportActionsList.js

```diff
@@ -152,6 +152,16 @@ function ReportActionsList({
         opacity: opacity.value,
     }));
 
+    // This hook fixes the position of the FlatList when the user is at the bottom of the list and returns from other screens after deleting list items
+    useFocusEffect(
+        useCallback(() => {
+            if (scrollingVerticalOffset.current !== 0) {
+                return;
+            }
+            reportScrollManager.scrollToBottom();
+        }, [reportScrollManager]),
+    );
```

Comment:
This does feel like a workaround. I also remember this bug being related to the use of Freeze or is this bug related only to this PR (i.e. not reproducible in main)?

Reply:
It is connected with the fact that now we go back to the screen which is already on the navigation stack. On the main branch we invoke two navigate functions after delete the request money. In the result we render flatList for the first time and this bug doesn't occur. Now instead of pushing new screen to the navigation stack we go back to the existing one. After the deletion process the flatList doesn't automatically scroll to the bottom, there is a free space left after the deleted item. This hook solves this problem

Reply:
I removed the hook and was not able to reproduce. Can you please double check?


https://github.com/Expensify/App/assets/16493223/e7ba8a7e-85bf-4995-8067-f10eccab933c

Reply:
Could you check it on the mobile platforms? On web this bug doesn't occur

Reply:
Same 

https://github.com/Expensify/App/assets/16493223/20f80554-8c82-4357-a34f-c0cb9ee6f843

Reply:
https://github.com/Expensify/App/assets/47774969/9f2c6732-03e9-41c2-a968-bdf661a469ce

I've checked it again and I am able to reproduce it. Before the test I pulled the last changes from main

Reply:
The bug is only reproducible on iOS


https://github.com/Expensify/App/assets/16493223/8c25aa3c-bc8f-49a9-8015-babf395917f7

Reply:
I think the root cause is the autoscrollToTopThreshold value in maintainVisibleContentPosition